staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/validation/validation.go

```diff
@@ -358,3 +361,31 @@ func isValidConditionReason(value string) []string {
 	}
 	return nil
 }
+
+// ValidateIgnoreStoreReadError validates that delete options are valid when
+// ignoreStoreReadErrorWithClusterBreakingPotential is enabled
+func ValidateIgnoreStoreReadError(fldPath *field.Path, options *metav1.DeleteOptions) field.ErrorList {
```

Comment:
nothing at the moment, it will have no effect, do we want me to add a validation inside `DeleteCollection` handler?

Reply:
> nothing at the moment, it will have no effect, do we want me to add a validation inside `DeleteCollection` handler?

We should ensure the value isn't set for DeleteCollection calls.

Reply:
yes, validation added